assets/dev/scss/frontend/widgets/playing-cards.scss

```
@@ -0,0 +1,211 @@
+.e-playing-cards-container {
```

I think we can use the main widget wrapper class, then we can simplify the selectors inside the controls.

Something like this:
```suggestion
.elementor-widget-playing-cards {
```

---

Done